mypy.ini

```diff
@@ -1,6 +1,2 @@
 [mypy]
```

Comment:
Why did you need to pass the options explicitly in noxfile.py? This will break direct usage of `mypy` in the future.

Now, I suppose there is a valid reason, which brings me to my next question: why keep this file at all?

Reply:
Oh, this was just a matter of preference, and to be consistent with other tools that have their options in the noxfile. I will remove this file.